pkg/manager/manager_test.go

```diff
@@ -0,0 +1,191 @@
+	type args struct {
+		listOfImageOrVersionOrPRs []string
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "Empty arguments: 'launch'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{""},
+			},
+			want: false,
+		},
+		{
+			name: "Valid version by itself: 'launch 4.19'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"4.19"},
+			},
+			want: true,
+		},
+		{
+			name: "Valid nightly version by itself: 'launch 4.19-nightly'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"4.19"},
+			},
+			want: true,
+		},
+		{
+			name: "Valid dot nightly version by itself: 'launch 4.19.nightly'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"4.19"},
+			},
+			want: true,
+		},
+		{
+			name: "Valid nightly version with 6 digit tail by itself: 'launch 4.20.0-0.nightly-2025-04-02-081925'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"4.20.0-0.nightly-2025-04-02-081925"},
+			},
+			want: true,
+		},
+		{
+			name: "Valid latest nightly version by itself : 'launch 4.20.0-0.nightly'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"4.20.0-0.nightly"},
+			},
+			want: true,
+		},
+		{
+			name: "Valid nightly version with 6 digit tail by itself: 'launch 4.20.0-0.nightly-2025-04-02-081925'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"4.20.0-0.nightly-2025-04-02-081925"},
+			},
+			want: true,
+		},
+
+		{
+			name: "Valid latest ci version by itself : 'launch 4.20.0-0.ci'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"4.20.0-0.ci"},
+			},
+			want: true,
+		},
+		{
+			name: "Valid ci version with 6 digit tail by itself: 'launch 4.20.ci-2025-04-02-081925'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"4.20.ci-2025-04-02-081925"},
+			},
+			want: true,
+		},
+		{
+			name: "Valid ci version by itself: 'launch 4.20.ci'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"4.20.ci"},
+			},
+			want: true,
+		},
+		{
+			name: "Valid nightly registry version by itself: 'launch registry.ci.openshift.org/ocp/release:4.20.0-0.nightly-2025-04-02-081925'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"registry.ci.openshift.org/ocp/release:4.20.0-0.nightly-2025-04-02-081925"},
+			},
+			want: true,
+		},
+		{
+			name: "Valid ci registry version by itself: 'launch registry.ci.openshift.org/ocp/release:4.20.0-0.ci-2025-04-02-081925'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"registry.ci.openshift.org/ocp/release:4.20.0-0.ci-2025-04-02-081925"},
+			},
+			want: true,
+		},
+		{
+			name: "Using a pull request without version specified: 'launch openshift/installer#7160'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"openshift/installer#7160"},
+			},
+			want: false,
+		},
+		{
+			name: "Using a pull request with a version specified: 'launch 4.19,openshift/installer#7160'",
+			args: args{
+				listOfImageOrVersionOrPRs: []string{"4.19,openshift/installer#7160"},
```

Comment:
This should be a slice of 2 values: `[]string{"4.19", "openshift/installer#7160"}`

Reply:
yes, let me fix -- thanks. Also, the optional comma is not necessary because by the time the `containsValidVersion` func is called, the `listOfImageOrVersionOrPRs` is already a slice (without commas) -- I'm removing the optional comma from the regex.